charts/tm-forum-api/values.yaml

```diff
@@ -0,0 +1,233 @@
+    # ingress.kubernetes.io/ssl-redirect: "true"
+  # -- all hosts to be provided
+  hosts: 
+    # -- provide a hosts and the paths that should be available
+    - host: localhost
+      # paths:
+      # - /
+  # -- configure the ingress' tls
+  tls: []
+    # - secretName: tmforum-tls
+      # hosts:
+        # - tmforum.fiware.org
+
+## configuration to be used by every api-deployment if nothing specific is provided.
+defaultConfig:
+   # -- additional labels for the deployment, if required
+  additionalLabels: {}
+  # -- additional annotations for the deployment, if required
+  additionalAnnotations: {}
+  # -- selector template
+  # ref: https://kubernetes.io/docs/user-guide/node-selection/
+  nodeSelector: {}
+  # -- tolerations template
+  # ref: ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  tolerations: []
+  # -- affinity template
+  # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  affinity: {}
+  # -- additional sidecars for the deployment, if required
+  sidecars: []
+  # -- initial number of target replications, can be different if autoscaling is enabled
+  replicaCount: 1
+  # -- number of old replicas to be retained
+  revisionHistoryLimit: 3
+  ## configuration of the tmforum update strategy
+  updateStrategy:
+    # -- type of the update
+    type: RollingUpdate
+    # -- new pods will be added gradually
+    rollingUpdate:
+      # -- number of pods that can be created above the desired amount while updating
+      maxSurge: 1
+      # -- number of pods that can be unavailable while updating
+      maxUnavailable: 0
+
+  # -- tmforum resource requests and limits, we leave the default empty to make that a concious choice by the user.
+  # for the autoscaling to make sense, you should configure this.
+  # resources:
+    # limits:
+      # cpu: 100m
+      # memory: 128Mi
+    # requests:
+      # cpu: 100m
+      # memory: 128Mi
+  ## liveness and readiness probes of the tmforum api
+  # ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  # if one of the values should be overwritten by the individual api, you need to provide all values.
+  livenessProbe:
+    # -- path to be used for the health check
+    healthPath: /health
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 30
+  readinessProbe:
+    # -- path to be used for the health check
+    readinessPath: /health
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 30
+   
+  # -- a list of additional env vars to be set, check the tm-forum api docu for all available options
+  additonalEnvVars: []
+
+  # -- configuration to be used for the image of the container
+  image:
+    # -- repository to get the container from
+    repository: quay.io/wi_stefan
+    # -- tag to be used, most of the time the apis will use the same version
+    tag: 0.4.1
```

Comment:
Do all images always have the same version tag?

Reply:
Not neccessarily, thats why you can overwrite them individually

Reply:
I did overlook that. The added description in the Readme is good.